src/App.js

```diff
@@ -64,7 +64,7 @@ const userIsLoggedInButMustCompleteJoinForm = loggedInUser => {
 };
 
 const forceSelectRole = ({ loggedInUser, isLoadingUser, WrapperPage = Page, ...props }) => {
-  if (userIsRequiredToLogIn(loggedInUser)) {
+  if (userIsRequiredToLogIn(loggedInUser) && !isLoadingUser) {
     return isLoadingUser ? null : (
```

Comment:
isLoadingUser at line 67 will always be false. I think we should remove the condition.

Reply:
it could be true, depending on what happen on the freactal initialization.

Reply:
I meant : because of the condition the 'isLoadingUser' at line 68 will never evaluate to to true at that line since the if condition above prevents it.

Reply:
fixed